### DISCUSSION TITLE
SONOFF ZBBridge Vs ZBdongle

### DISCUSSION THREAD

**Discussion by nromagna** [indomus/forum#2346] [HUB: Home Assistant]
Ciao a tutti, al momento ho un normalissimo ZBBridge (scatolina bianca) che diciamo funziona bene anche se in alcuni angoli della casa il segnale è debole.
Devo ammettere che non ho molti Router ZB in giro per casa, la maggior parte sono solo sensori...

Quindi gentilmente chiedo se cambiassi il mio Bridge con il ZBDongle (poi al massimo mi aiutate anche a capire la differenza tra il modello con e senza "E" finale..):
1. il segnale migliora?;
2. lo posso, un volta configurato, alimentare NON da Raspberry (dove ho HA) ma da presa USB attaccata a muro?;
3. devo riconfigiurare tutti i dispositivi ZB?

Grazie.

**Comment by indomus** [comment]
> Ciao a tutti, al momento ho un normalissimo ZBBridge (scatolina bianca) che diciamo funziona bene anche se in alcuni angoli della casa il segnale è debole. Devo ammettere che non ho molti Router ZB in giro per casa, la maggior parte sono solo sensori...
> 
> Quindi gentilmente chiedo se cambiassi il mio Bridge con il ZBDongle (poi al massimo mi aiutate anche a capire la differenza tra il modello con e senza "E" finale..):
> 
> 1. il segnale migliora?;
> 2. lo posso, un volta configurato, alimentare NON da Raspberry (dove ho HA) ma da presa USB attaccata a muro?;
> 3. devo riconfigiurare tutti i dispositivi ZB?
> 
> Grazie.

ciao!
1) sì. sono coordinator decisamente più potenti.
2) sì, basta un usb, se le usi come router. se le usi come coordinator ti serve un bridge/gaway sw di gestione (zha, zigbee2mqtt o quel che ti pare) - e quindi un host, come raspberry o altro
3) sì

la -E è la versione col chipset più moderno, matter ready. la -P di contro è più supportata dai più comuni bridge/gateway sw come quelli sopracitati.

**Reply by nromagna** [reply]
Ciao, intanto grazie per la rapida risposta, poi

1. ottima notizia.
2/a. In che senso come router? Posso lasciare il ZBBridge "vecchio" funzionante ed utilizzare questo solo come router?
2/b. Mentre per utilizzare il software zha, zigbee2mqtt lo installo in HA giuto? E il ZBdongle lo collego alla usb del raspberry?
3. ok me li ripasso con calma.
cosa consigli? -E o -P?
Grazie

**Reply by KC-inDomus** [reply]
> Ciao, intanto grazie per la rapida risposta, poi
> 
> 1. ottima notizia.
>    2/a. In che senso come router? Posso lasciare il ZBBridge "vecchio" funzionante ed utilizzare questo solo come router?
>    2/b. Mentre per utilizzare il software zha, zigbee2mqtt lo installo in HA giuto? E il ZBdongle lo collego alla usb del raspberry?
> 2. ok me li ripasso con calma.
>    cosa consigli? -E o -P?
>    Grazie

ok, ti mancano un po' di dettagli. [leggi qui](https://indomus.it/componenti/zigbee/) con grande attenzione, dovrebbe fartisi un quadro chiaro in testa.

**Reply by nromagna** [reply]
OK inizia ad essere un pochino più chiaro.

Il vecchio ZBBridge lo spengo prima di fare qualsiasi cosa! Devo fare lo sresso anche con quello Hue? E nel caso poi riconfigurare TUTTE le lampadine Hue di casa? 

Diciamo che a questo punto _mi inspira_ ZigBee2MQTT. Al momento a casa è al 50% con Sonoff, 40% Philips e resto qualcosa (3 lampadine Tuya e 1 lampada Yeelight)... quindi per partire punto al -E visto che sembra la più recente oppure rimango sulla -P visto i dipositivi sopra?

Grazie.

**Reply by KC-inDomus** [reply]
ti ispira giusto imho.
i precedenti bidge li puoi anche pensioare, io uso sia la -P che la -E su scenari diversi, certamente tra le due la -E è la più moderna

**Reply by nromagna** [reply]
Oridnato (ho beccato anche un piccolo sconto su AZ, nel caso posso interessare).

Ricapitolando tolgo i bidge Sonoff e Hue, attacco il nuovo al Raspberry (dove gira HA) e da li installo ZigBee2MQTT e tutto il resto, quindi riconfiguro tutti i dispositivi.

Poi un domani nel caso posso acquistare un secondo ZBdongle attaccarlo ad una presa USB ed utilizzarlo come router, giusto?

**Reply by indomus** [reply]
yes

**Reply by nromagna** [reply]
Ultima richiesta, ma se sostituisco i due bridge (Hue + Sonoff) con il nuovo poi come le controllo i dispositivi con Google/Alexa?
Grazie.

**Reply by nromagna** [reply]
Nessuno aiutino? Si insomma presumo la risposta sia il servizio a pagamento... ma se facessi (per il momento) un passo indietro ed volessi utilizzare Dongle SOLO come router, come lo configuro? Grazie,

**Reply by indomus** [reply]
gli installi il firmware router e lo associ alla rete, tutto qui
